easylistdutch/hide_whitelist.txt

```diff
@@ -1,35 +1,37 @@
 autodealers.nl,speurders.nl,xxxplaza.nl#@##advertentie
-drenthemobiel.nl,flevolandmobiel.nl,gelderlandmobiel.nl,groningenmobiel.nl,limburgmobiel.nl,marktplaza.nl,moorslede.be,nederlandmobiel.nl,noord-brabantmobiel.nl,noord-hollandmobiel.nl,overijsselmobiel.nl,paginamarkt.nl,sexjobs.nl,utrechtmobiel.nl,zeelandmobiel.nl,zuid-hollandmobiel.nl#@#.advertentie
 blikopnieuws.nl,computeridee.nl#@#.reclame
+drenthemobiel.nl,flevolandmobiel.nl,gelderlandmobiel.nl,groningenmobiel.nl,limburgmobiel.nl,marktplaza.nl,moorslede.be,nederlandmobiel.nl,noord-brabantmobiel.nl,noord-hollandmobiel.nl,overijsselmobiel.nl,paginamarkt.nl,sexjobs.nl,utrechtmobiel.nl,zeelandmobiel.nl,zuid-hollandmobiel.nl#@#.advertentie
 ! Uitzonderingen voor EasyList
-innerlijk-besef.nl#@##ad-main
-xxxplaza.nl#@##ad_right
-autobedrijfscheele.nl,autodealers.nl,autoroden.nl,henkheikamp.nl#@##ad_wrap
-xxxplaza.nl#@##adcontainer
-kinderspelletjes.nl#@##adframe:not(frameset)
-comedycentral.nl#@##bannerad
+ad.nl#@#.ad_bg
+ad.nl#@#.nav_ad
 archiefleeuwardercourant.nl,dekrantvantoen.nl#@##headerAdv
-schaatsen.nl#@#.HeaderAd
-innerlijk-besef.nl,landleven.nl#@#.ad-block
-bokt.nl#@#.ad-body
+autobedrijfscheele.nl,autodealers.nl,autoroden.nl,henkheikamp.nl#@##ad_wrap
+autobedrijfscheele.nl,autodealers.nl,autoroden.nl,henkheikamp.nl#@#.ad_background
 autodealers.nl#@#.ad-btn
-innerlijk-besef.nl#@#.ad-button
-innerlijk-besef.nl#@#.ad-footer
-innerlijk-besef.nl#@#.ad-header
 bd.nl,bndestem.nl,destentor.nl,ed.nl,gelderlander.nl,pzc.nl,radiozenders.fm,tubantia.nl#@#.ad-spacer
-marktplaats.nl#@#.adSummary
+bokt.nl#@#.ad-body
+bokt.nl#@#.node-ad
+comedycentral.nl#@##bannerad
 digitekst.com#@#.adTitle
-autobedrijfscheele.nl,autodealers.nl,autoroden.nl,henkheikamp.nl#@#.ad_background
-ad.nl#@#.ad_bg
-speurders.nl#@#.ad_header
+emarketingexperts.nl#@#.adwords
 forum.fok.nl#@#.adfieldbg
 hotcontact.nl,xxxplaza.nl#@#.adlink
-zeelandnet.nl#@#.ad-card
-schaatsen.nl#@#.advertisement-header
-emarketingexperts.nl#@#.adwords
-ad.nl#@#.nav_ad
-bokt.nl#@#.node-ad
-landleven.nl#@#.post-ad
 kamernet.nl#@#.topAdd
+kinderspelletjes.nl#@##adframe:not(frameset)
+landleven.nl#@#.post-ad
+marktplaats.nl#@#.adSummary
+schaatsen.nl#@#.advertisement-header
+schaatsen.nl#@#.HeaderAd
+spele.nl#@#.afs_ads
+speurders.nl#@#.ad_header
+vakantieplaats.nl#@#.feed-ad
+vakantieplaats.nl#@#.topAds
 voetbalnieuws.be#@#a[href^="http://adserving.unibet.com/"]
-tweakers.net#@#a[href^="http://pubads.g.doubleclick.net/"]
+wereldklokken.nl#@#.adsbygoogle
+xxxplaza.nl#@##ad_right
+xxxplaza.nl#@##adcontainer
+zeelandnet.nl#@#.ad-card
+!Anti-Adblock
+directwonen.nl#@#.adsbox
+fok.nl#@#.advertentie
+forum.fok.nl#@##advertentie
```

Comment:
De laatste is overbodig. daar je al op het hele domain #@#.advertentie toegepast hebt.

Reply:
> De laatste is overbodig. daar je al op het hele domain #@#.advertentie toegepast hebt.

fok.nl is .advertentie
forum.fok.nl heeft geen . maar een #